### DISCUSSION TITLE
Can I remove the users I blocked???

### DISCUSSION THREAD

**Discussion by shanliu** [community/community#16175] [Other Feature Feedback, Questions, & Ideas]
Can I remove the users I blocked
In ""Recommended based on people you follow""

**Comment by edhends** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report

**Comment by MathiasReker** [comment]
No this is not possible

**Comment by siminapasat** [comment]
@shanliu what do you understand by "remove the user"? Where do you want to remove them from? Or do you want to undo the blocking action?

**Reply by jfullstackdev** [reply]
I think he is referring to the feed, what I'm seeing is, the user already blocked them so as not to be seen on the feed, but still they are on the feed

**Reply by siminapasat** [reply]
This shouldn't be the case (and if it is, it's a bug we should be fixing!), @shanliu can you please clarify what you are seeing? Thank you :)

**Reply by shanliu** [reply]
> I think he is referring to the feed, what I'm seeing is, the user already blocked them so as not to be seen on the feed, but still they are on the feed

YES...

**Comment by itzmail** [comment]
remove user is mean that user is not on list or your feed right if it's like that, just unfollow that person

**Reply by shanliu** [reply]
I didn't follow him

**Comment by jfullstackdev** [comment]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182015733-3ede7b72-71b8-4ddf-bbb7-4826480973ba.png)

**Reply by shanliu** [reply]
thanks!!
